src/QueryDescription/index.js

```diff
@@ -51,8 +50,30 @@ export type On = $RE<{
   left: ColumnName,
   comparison: Comparison,
 }>
-export type Condition = Where | On
-export type QueryDescription = $RE<{ where: Where[], join: On[] }>
+export type SortOrder =
+  | 'asc'
+  | 'desc'
```

Comment:
maybe we can export those so that you can use Q.asc, Q.desc and not worry about a mistake?